src/instructlab/model/train.py

```diff
@@ -473,22 +493,18 @@ def get_files(directory: str, pattern: str) -> list[str]:
         shutil.copy(train_files[0], train_file)
         shutil.copy(test_files[0], test_file)
 
-    if (
-        pipeline == "full"
-        or pipeline == "accelerated"
-        or strategy == SupportedTrainingStrategies.LAB_MULTIPHASE.value
-    ):
+    if pipeline == "full":
         if not os.path.isfile(data_path):
             ctx.fail(
                 f"Data path must be to a valid .jsonl file. Value given: {data_path}"
             )
 
         # pull the trainrandom.randinting and torch args from the flags
         # the flags are populated from the config as a base.
-        train_args, torch_args = map_train_to_library(ctx, ctx.params)
-        logger.debug(
-            "Rendered training arguments:\n%s", pprint.pformat(train_args.model_dump())
-        )
+    train_args, torch_args = map_train_to_library(ctx, ctx.params)
```

Comment:
this also needs to be indented, I think this will get triggered for simple, I might be reading this wrong

Reply:
got it